packages/design-system/src/Tooltip/Tooltip.css

```diff
@@ -0,0 +1,44 @@
+/* TODO: Find a better way to style */
```

Comment:
Can you please add comments on why we can't use our traditional method, along with what each part of the class is for?